src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -292,8 +303,26 @@ const DataAccessRequestApplication = (props) => {
 
     batchFormFieldChange(formData);
     window.addEventListener('scroll', onScroll); // eslint-disable-line -- codacy says event listeners are dangerous
+    setIsLoading(false);
```

Comment:
Moving setIsLoading to later in this init method so that the dar code and project title are loaded to be displayed as the header before we remove the loading spinner.